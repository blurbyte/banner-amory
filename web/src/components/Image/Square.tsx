import styled from 'styled-components';

const Square = styled.div`
  width: 100%;
  padding-bottom: 100%;
  position: relative;
`;

export default Square;
