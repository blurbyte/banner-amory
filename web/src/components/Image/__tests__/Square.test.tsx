import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import Square from '../Square';

test('renders correctly', () => {
  const component = renderer.create(<Square />);
  expect(component.toJSON()).toMatchSnapshot();
});
