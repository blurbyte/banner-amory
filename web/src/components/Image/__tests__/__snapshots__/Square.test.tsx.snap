// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  width: 100%;
  padding-bottom: 100%;
  position: relative;
}

<div
  className="c0"
/>
`;
